src/transaction/log_applier_sql_log.c

```diff
@@ -546,6 +562,11 @@ sl_write_sql (string_buffer & query, string_buffer * select)
   if (ftell (log_fp) >= SL_LOG_FILE_MAX_SIZE)
     {
       log_fp = sl_open_next_file (log_fp);
+
+      if (sql_log_keep_cnt != 0 && sl_Info.curr_file_id >= sql_log_keep_cnt)
+	{
+	  sl_remove_oldest_file ();
```

Comment:
```suggestion
	  (void) sl_remove_oldest_file ();
```
It might cause a warning. `sl_remove_oldest_file()` really needs to return an error code, by the way?

Reply:
Or, if we should keep it returning an error code, I think we should do error handling even though now `sl_remove_oldest_file` always returns NO_ERROR. In my opinion, the caller should not expect or assume the return value of a callee.
```suggestion
	  if (sl_remove_oldest_file () != NO_ERROR)
	  {
	    return ER_FAILED;
	  }
```